template/module/README.rst

```diff
@@ -65,8 +65,8 @@ Images
 Contributors
 ------------
 
-* Firstname Lastname <email.address@example.org>
-* Second Person <second.person@example.org>
+* Firstname Lastname <email.address@example.org> (optional company website url)
```

Comment:
A little strange to me to mix contributors (=usually individuals) with companies.
Could we think about separating or providing better visibility to the companies? Honestly funding and money comes most of the time from customers or integrators.